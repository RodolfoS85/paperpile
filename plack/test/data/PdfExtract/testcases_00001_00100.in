---
file: ../../../PDFREPOSITORY/000001.pdf
---
file: ../../../PDFREPOSITORY/000002.pdf
---
file: ../../../PDFREPOSITORY/000003.pdf
---
file: ../../../PDFREPOSITORY/000004.pdf
---
file: ../../../PDFREPOSITORY/000005.pdf
---
file: ../../../PDFREPOSITORY/000006.pdf
---
file: ../../../PDFREPOSITORY/000007.pdf
---
file: ../../../PDFREPOSITORY/000008.pdf
---
file: ../../../PDFREPOSITORY/000009.pdf
---
file: ../../../PDFREPOSITORY/000010.pdf
---
file: ../../../PDFREPOSITORY/000011.pdf
---
file: ../../../PDFREPOSITORY/000012.pdf
---
file: ../../../PDFREPOSITORY/000013.pdf
---
file: ../../../PDFREPOSITORY/000014.pdf
---
file: ../../../PDFREPOSITORY/000015.pdf
---
file: ../../../PDFREPOSITORY/000016.pdf
---
file: ../../../PDFREPOSITORY/000017.pdf
---
file: ../../../PDFREPOSITORY/000018.pdf
---
file: ../../../PDFREPOSITORY/000019.pdf
---
file: ../../../PDFREPOSITORY/000020.pdf
---
file: ../../../PDFREPOSITORY/000021.pdf
---
file: ../../../PDFREPOSITORY/000022.pdf
---
file: ../../../PDFREPOSITORY/000023.pdf
---
file: ../../../PDFREPOSITORY/000024.pdf
---
file: ../../../PDFREPOSITORY/000025.pdf
---
file: ../../../PDFREPOSITORY/000026.pdf
---
file: ../../../PDFREPOSITORY/000027.pdf
---
file: ../../../PDFREPOSITORY/000028.pdf
---
file: ../../../PDFREPOSITORY/000029.pdf
---
file: ../../../PDFREPOSITORY/000030.pdf
---
file: ../../../PDFREPOSITORY/000031.pdf
---
file: ../../../PDFREPOSITORY/000032.pdf
---
file: ../../../PDFREPOSITORY/000033.pdf
---
file: ../../../PDFREPOSITORY/000034.pdf
---
file: ../../../PDFREPOSITORY/000035.pdf
---
file: ../../../PDFREPOSITORY/000036.pdf
---
file: ../../../PDFREPOSITORY/000037.pdf
---
file: ../../../PDFREPOSITORY/000038.pdf
---
file: ../../../PDFREPOSITORY/000039.pdf
---
file: ../../../PDFREPOSITORY/000040.pdf
---
file: ../../../PDFREPOSITORY/000041.pdf
---
file: ../../../PDFREPOSITORY/000042.pdf
---
file: ../../../PDFREPOSITORY/000043.pdf
---
file: ../../../PDFREPOSITORY/000044.pdf
---
file: ../../../PDFREPOSITORY/000045.pdf
---
file: ../../../PDFREPOSITORY/000046.pdf
---
file: ../../../PDFREPOSITORY/000047.pdf
---
file: ../../../PDFREPOSITORY/000048.pdf
---
file: ../../../PDFREPOSITORY/000049.pdf
---
file: ../../../PDFREPOSITORY/000050.pdf
---
file: ../../../PDFREPOSITORY/000051.pdf
---
file: ../../../PDFREPOSITORY/000052.pdf
---
file: ../../../PDFREPOSITORY/000053.pdf
---
file: ../../../PDFREPOSITORY/000054.pdf
---
file: ../../../PDFREPOSITORY/000055.pdf
---
file: ../../../PDFREPOSITORY/000056.pdf
---
file: ../../../PDFREPOSITORY/000057.pdf
---
file: ../../../PDFREPOSITORY/000058.pdf
---
file: ../../../PDFREPOSITORY/000059.pdf
---
file: ../../../PDFREPOSITORY/000060.pdf
---
file: ../../../PDFREPOSITORY/000061.pdf
---
file: ../../../PDFREPOSITORY/000062.pdf
---
file: ../../../PDFREPOSITORY/000063.pdf
---
file: ../../../PDFREPOSITORY/000064.pdf
---
file: ../../../PDFREPOSITORY/000065.pdf
---
file: ../../../PDFREPOSITORY/000066.pdf
---
file: ../../../PDFREPOSITORY/000067.pdf
---
file: ../../../PDFREPOSITORY/000068.pdf
---
file: ../../../PDFREPOSITORY/000069.pdf
---
file: ../../../PDFREPOSITORY/000070.pdf
---
file: ../../../PDFREPOSITORY/000071.pdf
---
file: ../../../PDFREPOSITORY/000072.pdf
---
file: ../../../PDFREPOSITORY/000073.pdf
---
file: ../../../PDFREPOSITORY/000074.pdf
---
file: ../../../PDFREPOSITORY/000075.pdf
---
file: ../../../PDFREPOSITORY/000076.pdf
---
file: ../../../PDFREPOSITORY/000077.pdf
---
file: ../../../PDFREPOSITORY/000078.pdf
---
file: ../../../PDFREPOSITORY/000079.pdf
---
file: ../../../PDFREPOSITORY/000080.pdf
---
file: ../../../PDFREPOSITORY/000081.pdf
---
file: ../../../PDFREPOSITORY/000082.pdf
---
file: ../../../PDFREPOSITORY/000083.pdf
---
file: ../../../PDFREPOSITORY/000084.pdf
---
file: ../../../PDFREPOSITORY/000085.pdf
---
file: ../../../PDFREPOSITORY/000086.pdf
---
file: ../../../PDFREPOSITORY/000087.pdf
---
file: ../../../PDFREPOSITORY/000088.pdf
---
file: ../../../PDFREPOSITORY/000089.pdf
---
file: ../../../PDFREPOSITORY/000090.pdf
---
file: ../../../PDFREPOSITORY/000091.pdf
---
file: ../../../PDFREPOSITORY/000092.pdf
---
file: ../../../PDFREPOSITORY/000093.pdf
---
file: ../../../PDFREPOSITORY/000094.pdf
---
file: ../../../PDFREPOSITORY/000095.pdf
---
file: ../../../PDFREPOSITORY/000096.pdf
---
file: ../../../PDFREPOSITORY/000097.pdf
---
file: ../../../PDFREPOSITORY/000098.pdf
---
file: ../../../PDFREPOSITORY/000099.pdf
---
file: ../../../PDFREPOSITORY/000100.pdf
