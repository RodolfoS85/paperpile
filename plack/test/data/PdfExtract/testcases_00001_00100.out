---
authors: 'van der Meulen, Meindert and Smit, Jan'
doi: 10.1088/1475-7516/2007/11/023
title: Classical approximation to quantum cosmological correlations
---
authors: 'Abkevich, V. I. and Gutin, A. M. and Shakhnovich, E. I.'
title: COMPUTER SIMULATIONS OF PREBIOTIC EVOLUTION
---
authors: 'Abney, Steven P.'
title: Stochastic Attribute-Value Grammars
---
authors: 'Addess, Kenneth J. and Basilion, James P. and Klausner, Richard D. and Rouault, Tracey A. and Pardi, Arthur'
title: 'Structure and Dynamics of the Iron Responsive Element RNA: Implications for Binding of the RNA by Iron Regulatory Binding Proteins'
---
authors: 'Addo-Quaye, Charles and Miller, Webb and Axtell, Michael J.'
title: 'CleaveLand: A pipeline for using degradome data to find cleaved small RNA targets'
---
authors: 'Aebersold, Ruedi and Mann, Matthias'
title: Mass spectrometry-based proteomics
---
authors: 'Agrawal, Ritesh and Stormo, Gary D.'
doi: 10.1261/rna.7280605
title: Editing efficiency of a Drosophila gene correlates with a distant splice site selection
---
authors: 'Agresti, Alessandra and Bianchi, Marco E'
doi: 10.1016/S0959-437X0300023-6
title: HMGB proteins and gene expression
---
authors: 'Aguirre-Hernández, Rosalía and Hoos, Holger H and Condon, Anne'
doi: 10.1186/1471-2105-8-34
title: 'Computational RNA secondary structure design: empirical complexity and improved methods'
---
authors: 'Akiva, Pinchas and Toporik, Amir and Edelheit, Sarit and Peretz, Yifat and Diber, Alex and Shemesh, Ronen and Novik, Amit and Sorek, Rotem'
doi: 10.1101/gr.4137606
title: Transcription-mediated gene fusion in the human genome
---
authors: 'Akutsu, Tatsuya and Halldrsson, Magns M.'
title: On the approximation of largest common subtrees and largest common point sets
---
authors: 'Akutsu, Tatsuya'
title: Dynamic programming algorithms for RNA secondary structure prediction with pseudoknots
---
authors: 'Akutsu, Tatsuya and Horimoto, Katsuhisa'
title: 'Local Multiple Alignment of Numerical Sequences: Detection of Subtle Motifs from Protein Sequences and Structures'
---
authors: 'Al-Lazikani, Bissan and Jung, Joon and Xiang, Zhexin and Honig, Barry'
title: Protein structure prediction
---
authors: 'Lpez-Ortiz, Alejandro and Quimper, Claude-Guy and Tromp, John and van Beek, Peter'
title: A Fast and Simple Algorithm for Bounds Consistency of the AllDifferent Constraint
---
authors: 'Aliferis, C. F. and Tsamardinos, I. and Statnikov, A.'
title: LARGE-SCALE FEATURE SELECTION USING MARKOV BLANKET INDUCTION FOR THE PREDICTION OF PROTEIN-DRUG BINDING
---
authors: 'Allali, Julien and Sagot, Marie-France'
title: Novel Tree Edit Operations for RNA Secondary Structure Comparison
---
authors: 'Allen, Patrick and Collins, Brian and Brown, David and Hostomsky, Zdenek and Gold, Larry'
title: A Specific RNA Structural Motif Mediates High Affinity Binding by the HIV-1 Nucleocapsid Protein (NCp7)
---
authors: 'Allen, Jonathan E and Salzberg, Steven L'
doi: 10.1186/1748-7188-1-14
title: A phylogenetic generalized hidden Markov model for predicting alternatively spliced exons
---
authors: 'Allen, Edwards and Xie, Zhixin and Gustafson, Adam M. and Carrington, James C.'
doi: 10.1016/j.cell.2005.04.004
title: microRNA-Directed Phasing during Trans-Acting siRNA Biogenesis in Plants
---
authors: 'Allmang, Christine and Carbon, Philippe and Krol, Alain'
title: 'The SBP2 and 15.5 kD/Snu13p proteins share the same RNA binding domain: Identification of SBP2 amino acids important to SECIS RNA binding'
---
authors: 'Allouche, D. and de Givry, S. and Sanchez, M. and Schiex, T.'
title: Tagsnp selection using Weighted CSP and Russian Doll Search with Tree Decomposition
---
authors: 'Alm, E. and Baker, D.'
title: Prediction of protein-folding mechanisms from free-energy landscapes derived from native structures
---
authors: 'Alon, U. and Barkai, N. and Notterman, D. A. and Gish, K. and Ybarra, S. and Mack, D. and Levine, A. J.'
title: Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays
---
authors: 'Aloy, Patrick and Russell, Robert B.'
doi: 10.1073/pnas.092147999
title: Interrogating protein interaction networks through structural biology
---
authors: 'Altman, Russ B.'
title: 'Probabilistic Structure Calculations: A Three-Dimensional tRNA Structure from Sequence Correlation Data'
---
authors: 'Altman, Russ B. and Weiser, Bryn and Noller, Harry F.'
title: 'Constraint Satisfaction Techniques for Modeling Large Complexes: Application to the Central Domain of 16S Ribosomal RNA'
---
authors: 'Altschul, Stephen F. and Bundschuh, Ralf and Olsen, Rolf and Hwa, Terence'
title: The estimation of statistical parameters for local alignment score distributions
---
authors: 'Altschul, Stephen F. and Ericksont, Bruce W.'
title: 'Significance of Nucleotide Sequence Alignments: A Method for Random Sequence Permutation That Preserves Dinucleotide and Codon Usage1'
---
authors: 'Altschul, Stephen F. and Madden, Thomas L. and Schäffer, Alejandro A. and Zhang, Jinghui and Zhang, Zheng and Miller, Webb and Lipman, David J.'
title: 'Gapped BLAST and PSI-BLAST: a new generation of protein database search programs'
---
authors: 'Altuvia, Shoshy and Wagner, E. Gerhart H.'
title: Switching on and off with RNA
---
authors: 'Alvarez, Marta and Rhodes, Simon J. and Bidwell, Joseph P.'
doi: 10.1016/S0378-11190300627-9
title: 'Context-dependent transcription: all politics is local'
---
authors: 'Ambros, Victor and Chen, Xuemei'
doi: 10.1242/dev.002006
title: The regulation of genes and genomes by small RNAs
---
authors: 'Ambros, Victor'
title: The functions of animal microRNAs
---
authors: 'Ameres, Stefan Ludwig and Martinez, Javier and Schroeder, Renee'
doi: 10.1016/j.cell.2007.04.037
title: Molecular Basis for Target RNA Recognition and Cleavage by Human RISC
---
authors: 'Anderson, Kelsi L. and Dunman, Paul M.'
doi: 10.1155/2009/525491
title: 'Messenger RNA Turnover Processes in Escherichia coli, Bacillus subtilis, and Emerging Studies in Staphylococcus aureus'
---
authors: 'Anderson, N Leigh and Matheson, Alastair D and Steiner, Sandra'
title: 'Proteomics: applications in basic and applied biology'
---
authors: 'Andersson, Arne and Hagerupt, Torben and Nilsson, Stefan and Ramam, Rajeev'
title: Sorting in Linear Time?
---
authors: 'Andres, Vicente and Cervera, Margarita and Mahdavi, Vijak'
title: Determination of the Consensus Binding Site for MEF2 Expressed in Muscle and Brain Reveals Tissue-specific Sequence Constraints
---
authors: 'Anestål, Karin and Arner, Elias S. J.'
doi: 10.1074/jbc.M210733200
title: Rapid Induction of Cell Death by Selenium-compromised Thioredoxin Reductase 1 but Not by the Fully Active Enzyme Containing Selenocysteine
---
authors: 'Anfinsen, Christian B.'
title: Principles that Govern the Folding of Protein Chains
---
authors: 'Ankerst, Mihael and Kastenmüller, Gabi and Kriegel, Hans-Peter and Seidl, Thomas'
title: Nearest Neighbor Classification in 3D Protein Databases
---
authors: 'Antal, M. and Mougin, A. and Kis, M. and Boros, E. and Steger, G. and Jakab, G. and Solymosy, F. and Branlant, C.'
title: 'Molecular characterization at the RNA and gene levels of U3 snoRNA from a unicellular green alga, Chlamydomonas reinhardtii'
---
authors: 'Antoniotti, Marco and Policriti, Alberto and Ugel, Nadia and Mishra, Bud'
title: Model Building and Model Checking for Biochemical Processes
---
authors: 'Ao, Wanyuan and Gaudet, Jeb and Kent, W. James and Muttumu, Srikanth and Mango, Susan E.'
title: Environmentally Induced Foregut Remodeling by PHA-4/FoxA and DAF-12/NHR
---
authors: 'Apic, Gordana and Gough, Julian and Teichmann, Sarah A.'
title: An insight into domain combinations
---
authors: 'Apweiler, R. and Attwood, T. K. and Bairoch, A. and Bateman, A. and Birney, E. and Biswas, M. and Bucher, P. and Cerutti, L. and Corpet, F. and Croning, M. D. R. and Durbin, R. and Falquet, L. and Fleischmann, W. and Gouzy, J. and Hermjakob, H. and Hulo, N. and Jonassen, I. and Kahn, D. and Kanapin, A. and Karavidopoulou, Y. and Lopez, R. and Marx, B. and Mulder, N. J. and Oinn, T. M. and Pagni, M. and Servant, F. and Sigrist, C. J. A. and Zdobnov, E. M.'
title: 'The InterPro database, an integrated documentation resource for protein families, domains and functional sites'
---
authors: 'Argaman, Liron and Altuvia, Shoshy'
doi: 10.1006/jmbi.2000.3942
title: 'fhlA Repression by OxyS RNA: Kissing Complex Formation at Two Sites Results in a Stable Antisense-Target RNA Complex'
---
authors: 'Argaman, Liron and Hershberg, Ruth and Vogel, Jorg and Bejerano, Gill and Wagner, E. Gerhart H. and Margalit, Hanah and Altuvia, Shoshy'
title: Novel small RNA-encoding genes in the intergenic regions of Escherichia coli
---
authors: 'Arndt, Peter F. and Burge, Christopher B. and Hwa, Terence'
title: DNA Sequence Evolution with Neighbor-Dependent Mutation
---
authors: 'Arner, Elias S. J. and Sarioglu, Hakan and Lottspeich, Friedrich and Holmgren, Arne and Bock, August'
title: 'High-level Expression in Escherichia coli of Selenocysteine-containing Rat Thioredoxin Reductase Utilizing Gene Fusions with Engineered Bacterial-type SECIS Elements and Co-expression with the selA, selB and selC Genes'
---
authors: 'Arnone, Maria I. and Davidson, Eric H.'
title: 'The hardwiring of development: organization and function of genomic regulatory systems'
---
authors: 'Arora, Sanjeev and Frieze, Alan and Kaplan, Haim'
title: A New Rounding Procedure for the Assignment Problem with Applications to Dense Graph Arrangement Problems
---
authors: 'Arratia, Richard and Bollobs, Bla and Coppersmith, Don and Sorkin, Gregory B.'
title: Euler circuits and DNA sequencing by hybridization
---
authors: 'Arslan, Abdullah N. and Egecioglu, Omer and Pevzner, Pavel A.'
title: 'A new approach to sequence comparison: normalized sequence alignment'
---
authors: 'Artzi, Shay and Kiezun, Adam and Shomron, Noam'
doi: 10.1186/1471-2105-9-39
title: 'miRNAminer: A tool for homologous microRNA gene search'
---
authors: 'JOE ARUSCAVAGE, P. and Bass, Brenda L.'
title: A phylogenetic analysis reveals an unusual sequence conservation within introns involved in RNA editing
---
authors: 'Aspnes, James and Hartling, Julia and Kao, MING-YANG and Kim, Junhyong and Shah, Gauri'
title: A Combinatorial Toolbox for Protein Sequence Design and Landscape Analysis in the Grand Canonical Model
---
authors: 'Auboeuf, Didier and Honig, Arnd and Berget, Susan M. and OMalley, Bert W.'
title: Coordinate Regulation of Transcription and Splicing by Steroid Receptor Coregulators
---
authors: 'Audit, Benjamin and Vaillant, Cedric and Arneodo, Alain and dAubenton-Carafa, Yves and Thermes, Claude'
doi: 10.1006/jmbi.2001.5363
title: 'Long-range Correlations between DNA Bending Sites: Relation to the Structure and Dynamics of Nucleosomes'
---
authors: 'Avilion, Ariel A. and Nicolis, Silvia K. and Pevny, Larysa H. and Perez, Lidia and Vivian, Nigel and Lovell-Badge, Robin'
doi: 10.1101/gad.224503
title: Multipotent cell lineages in early mouse development depend on SOX2 function
---
authors: 'Avner, Philip and Heard, Edith'
title: 'X-CHROMOSOME INACTIVATION: COUNTING, CHOICE AND INITIATION'
---
authors: 'Axmann, Ilka M and Kensche, Philip and Vogel, Jörg and Kohl, Stefan and Herzel, Hanspeter and Hess, Wolfgang R'
doi: 10.1186/gb-2005-6-9-r73
title: Identification of cyanobacterial non-coding RNAs by comparative genome analysis
---
authors: 'Babajide, Aderonke and Hofacker, Ivo L and Sippl, Manfred J and Stadler, Peter F'
title: 'Neutral networks in protein space: a computational study based on knowledge-based potentials of mean force'
---
authors: 'Bachellerie, Jean-Pierre and Cavaillé, Jérôme and Hüttenhofer, Alexander'
title: The expanding snoRNA world
---
authors: 'Backofen, Rolf and Gilbert, David'
title: Bioinformatics and Constraints
---
authors: 'Backofen, Rolf and Hermelin, Danny and Landau, Gad M. and Weimann, Oren'
title: Local Alignment of RNA Sequences with Arbitrary Scoring Schemes
---
authors: 'Backofen, Rolf and Landau, Gad M. and Mhl, Mathias and Tsur, Dekel and Weimann, Oren'
title: Fast RNA Structure Alignment for Crossing Input Structures
---
authors: 'Backofen, Rolf and Hermelin, Danny and Landau, Gad M. and Weimann, Oren'
title: Normalized Similarity of RNA Sequences
---
authors: 'Backofen, Rolf and Chen, Shihyen and Hermelin, Danny and Landau, Gad M. and Roytberg, Mikhail A. and Weimann, Oren and Zhang, Kaizhong'
doi: 10.1089/cmb.2007.0062
title: Locality and Gaps in RNA Comparison
---
authors: 'Backofen, Rolf and Will, Sebastian'
title: LOCAL SEQUENCE-STRUCTURE MOTIFS IN RNA
---
authors: 'Backofen, Rolf and Will, Sebastian'
title: A Constraint-Based Approach to Fast and Exact Structure Prediction in Three-Dimensional Protein Models
---
authors: 'Baek, Daehyun and Green, Phil'
doi: 10.1073/pnas.0506139102
title: 'Sequence conservation, relative isoform frequencies, and nonsense-mediated decay in evolutionarily conserved alternative splicing'
---
authors: 'Bafna, Vineet and Zhang, Shaojie'
title: 'FastR: Fast database search tool for non-coding RNA'
---
authors: 'Bagci, Zerrin'
doi: 10.1063/1.1432502
title: 'Residue packing in proteins: Uniform distribution on a coarse-grained scale'
---
authors: 'Bagci, Zerrin and Jernigan, Robert L. and Bahar, Ivet'
title: Residue coordination in proteins conforms to the closest packing of spheres
---
authors: 'Bahar, I. and Jernigan, R. L.'
title: Inter-residue Potentials in Globular Proteins and the Dominance of Highly Specific Hydrophilic Interactions at Close Separation
---
authors: 'Bahr, Anne and Thompson, Julie D. and -C. Thierry, J. and Poch, Olivier'
title: 'BAliBASE (Benchmark Alignment dataBASE): enhancements for repeats, transmembrane sequences and circular permutations'
---
authors: 'Bailey, Timothy L. and Gribskov, Michael'
title: 'Combining evidence using p-values: application to sequence homology searches'
---
authors: 'Bakheet, Tala and Frevel, Mathias and Williams, Bryan R. G. and Greer, William and Khabar, Khalid S. A.'
title: 'ARED: human AU-rich element-containing mRNA database reveals an unexpectedly diverse functional repertoire of encoded proteins'
---
authors: 'Bakkenist, Christopher J. and Kastan, Michael B.'
title: DNA damage activates ATM through intermolecular autophosphorylation and dimer dissociation
---
authors: 'Baldi, Pierre'
title: Periodic sequence patterns in human exons
---
authors: 'Ball, Stephen G. and Baldock, Clair and Kielty, Cay M. and Adrian Shuttleworth, C.'
title: The Role of the C1 and C2 A-domains in Type VI Collagen Assembly
---
authors: 'Bansal, Nikhil and Blum, Avrim and Chawla, Shuchi'
title: Correlation Clustering
---
authors: 'Bannai, Hideo and Tamada, Yoshinori and Maruyama, Osamu and Nakai, Kenta and Miyano, Satoru'
title: Extensive feature detection of N-terminal protein sorting signals
---
authors: 'Barash, Yoseph and Elidan, Gal and Friedman, Nir and Kaplan, Tommy'
title: Modeling Dependencies in Protein-DNA Binding Sites
---
authors: 'Barash, Y. and Elidan, G. and Kaplan, T. and Friedman, N.'
title: 'CIS: Compound Importance Sampling Method for Protein-DNA Binding Site p -value Estimation'
---
authors: 'Barbic, Andrej and Zimmer, Daniel P. and Crothers, Donald M.'
doi: 10.1073/pnas.0437877100
title: Structural origins of adenine-tract bending
---
authors: 'Barták, Roman'
title: Theory and Practice of Constraint Propagation
---
authors: 'Bartel, David P.'
title: 'MicroRNAs: Genomics, Biogenesis, Mechanism, and Function'
---
authors: 'Basler, Tina and Jeckstadt, Sabine and Valentin-Weigand, Peter and Goethe, Ralph'
doi: 10.1189/jlb.0905520
title: 'Mycobacterium paratuberculosis, Mycobacterium smegmatis, and lipopolysaccharide induce different transcriptional and post-transcriptional regulation of the IRG1 gene in murine macrophages'
---
authors: 'Bastien, Olivier and Maréchal, Eric'
doi: 10.1186/1471-2105-9-332
title: Evolution of biological sequences implies an extreme value distribution of type I for both global and local pairwise alignment scores
---
authors: 'Bastolla, U. and Vendruscolo, M. and Roman, H. E.'
title: 'Structurally constrained protein evolution: results from a lattice simulation'
---
authors: 'Bastolla, Ugo and Porto, Markus and Roman, H. Eduardo and Vendruscolo, Michele'
doi: 10.1007/s00239-002-2350-0
title: 'Connectivity of Neutral Networks, Overdispersion, and Structural Conservation in Protein Evolution'
---
authors: 'Bastolla, Ugo and Porto, Markus and Roman, H Eduardo and Vendruscolo, Michele'
doi: 10.1186/1471-2148-6-43
title: A protein evolution model with independent sites that reproduces site-specific amino acid distributions from the Protein Data Bank
---
authors: 'Bateman, Alex and Birney, Ewan and Cerruti, Lorenzo and Durbin, Richard and Etwiller, Laurence and Eddy, Sean R. and Griffiths-Jones, Sam and Howe, Kevin L. and Marshall, Mhairi and Sonnhammer, Erik L. L.'
title: The Pfam Protein Families Database
---
authors: 'Batzer, Mark A. and Deininger, Prescott L. and Hellmann-Blumberg, Utha and Jurka, Jerzy and Labuda, Damian and Rubin, Carol M. and Schmid, Carl W. and Zietkiewicz, Ewa and Zuckerkandl, Emile'
title: Standardized Nomenclature for Alu Repeats
---
authors: 'Bauer, Markus and Klau, Gunnar W. and Reinert, Knut'
title: Fast and Accurate Structural RNA Alignment by Progressive Lagrangian Optimization
---
authors: 'Bauer, Markus and Klau, Gunnar W.'
title: Structural Alignment of Two RNA Sequences with Lagrangian Relaxation
---
authors: 'Bauer, Markus and Klau, Gunnar W and Reinert, Knut'
doi: 10.1186/1471-2105-8-271
title: Accurate multiple sequence-structure alignment of RNA sequences using combinatorial optimization
